var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -159,25 +159,25 @@ def cmake_args(self):
 
         # Activate SIMD based on properties of the target
         target = self.spec.target
-        if target >= llnl.util.cpu.targets['zen2']:
+        if target >= archspec.cpu.TARGETS['zen2']:
```

Comment:
This is more of a TODO for later but it'd be nice if we could just compare `target` to a `str` here, and have the target do the lookup.